src/twisted/application/runner/_runner.py

```diff
@@ -31,19 +29,73 @@ class Runner(object):
     log = Logger()
 
 
-    def __init__(self, options):
+    def __init__(
+        self,
+        reactor=None,
+        pidFile=nonePIDFile, kill=False,
+        defaultLogLevel=LogLevel.info,
+        logFile=stderr, fileLogObserverFactory=textFileLogObserver,
+        whenRunning=lambda **_: None, whenRunningArguments={},
+        reactorExited=lambda **_: None, reactorExitedArguments={},
+    ):
         """
-        @param options: Configuration options for this runner.
-        @type options: mapping of L{RunnerOptions} to values
+        @param reactor: The reactor to start and run the application in.
+        @type reactor: L{IReactorCore}
+
+        @param pidFile: The file to store the running process ID in.
+        @type pidFile: L{IPIDFile}
+
+        @param kill: Whether this runner should kill an existing running
+            instance of the application.
+        @type kill: L{bool}
+
+        @param defaultLogLevel: The default log level to start the logging
```

Comment:
It's the default for new events.  See `LogLevelFilterPredicate.defaultLogLevel`.

What we're setting up here is the filter predicate so that you have a way to start using the namespace and log level filtering mechanisms that are not currently set up by `twistd` so you have to set that up yourself, and I imagine few people do.  What's not provided yet is a way to override the log level for a namespace, but that'll come later.

However, `twist` right now has a `--log-level` argument which uses to set the global default.